### DISCUSSION TITLE
Release v1.2.6 / 5.57.6

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#2057] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This build changes two core Sandboxie mechanics.
It reworks the hooking mechanism to enable proper unhooking on dll unload which should solve various compatibility issues.
It lets Sandboxie to use custom logon tokens instead of the anonymous one, the option is on by default and can be controlled with the "SandboxieLogon" setting.

This build also adds various UI improvements and a Vintage View Mode which makes Sandman.exe look almost like the classic SbieCtrl.exe.

# Known Issues
 - SSL connection fails with 3rd party anti malware tools when SSL scanning is enabled, the fix is to set `SandboxieLogon=n`
 https://github.com/sandboxie-plus/Sandboxie/issues/2025#issuecomment-1196343325

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

### Changed
- reworked saving of global options

### Fixed
- fixed issue with the Delete Content option [#2043](https://github.com/sandboxie-plus/Sandboxie/issues/2043)
- fixed issue with box prefs [#2046](https://github.com/sandboxie-plus/Sandboxie/issues/2046)
- fixed issue with Delete V2 registry

### Changed
- improved a few icons

### Fixed
- fixed certificate validation issue

### Added
- added delete button to the recovery window [#2024](https://github.com/sandboxie-plus/Sandboxie/issues/2024)

### Changed
- improved tree selection display

### Fixed
- fixed issues with file panel
- fixed issue with key bindings [#2030](https://github.com/sandboxie-plus/Sandboxie/issues/2030)
- fixed issue wtih RemoveSidName when terminating sbiesvc
- fixed issue with the new hooking mechanism
- fixed BSOD issue with win32k hooks introduced in 1.2.0 [#2035](https://github.com/sandboxie-plus/Sandboxie/issues/2035)
- fixed issue with element11 and electron workaround [#2023](https://github.com/sandboxie-plus/Sandboxie/issues/2023)

### Fixed
- fixed issues with new menu code

### Changed
- reworked frame drawing

### Fixed
- fixed issues with frame-drawing
- fixed issues with the tray and box menu introduced in the last build
- removed focus rectangle from the tree list

### Changed
- refactored menu creation code

### Added
- added Swedish translation to Plus UI (thanks pb1)
- added Vintage View Mode to make SandMan UI look like SbieCtrl
- added alternative tray menu mechanics
- added ability to auto-generate sandbox icons based on the border colour

### Changed
- changed box group icon to a dedicated one
- "browse content" is now available as a side panel in the main window
- animated hourglass icon overlay

### Fixed
- fixed DPI issue on Windows 7
- fixed issue with Software Compatibility tab
- fixed issue with OpenKeyPath introduced in build 1.1.1 [#2006](https://github.com/sandboxie-plus/Sandboxie/issues/2006)

### Added
- re-engineered "SandboxieLogon=y"; it's on by default, as every sandbox gets its own SID now
-- Note: this enforces strict isolation of sandboxes from each other.

### Changed
- reworked hook management, unloaded DLLs are properly unhooked now [#1243](https://github.com/sandboxie-plus/Sandboxie/issues/1243)
- box order is now stored in the Sandboxie-Plus.ini
- improved DPI scaling behaviour

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.2.6'>Release v1.2.6 / 5.57.6</a>.</em>

**Comment by BilleBarrett** [comment]
I have an extension  from IBM Security Rapport
Size: < 1 MBSize< 1 MBVersion: 2 . 2 . 1 2 7Version2.2.127
The client has to have it to do his banking with a certain Bank
PID 18608: SBIE2335 Initialization failed for process RapportHelper.exe [33 / 0]
How or where do I resolve this issue it happens in Chrome and Microsft Edge
Thanks  BilleBarrett
